### DISCUSSION TITLE
Version 1.22 of the app and Version 14 of the firmware

### DISCUSSION THREAD

**Discussion by fhdm-dev** [fhdm-dev/scoppy#92] [Announcements]
Version 1.22 of the app and Version 14 of the firmware were released a while ago but an announcement wasn't made because some of the most important new features were not documented at the time.

See the [Release Notes](https://oscilloscope.fhdm.xyz/wiki/ReleaseNotes) for a summary of the new features.

The [analog front end documentation](https://oscilloscope.fhdm.xyz/wiki/Analog-Front-End) has also been updated.

**Comment by mtanoushad24** [comment]
Firmware link unable to open
